provider/github/watcher.go

```diff
@@ -132,6 +175,29 @@ func (w *Watcher) handleEvent(r *lookout.RepositoryInfo, e *github.Event) (looko
 	return castEvent(r, e)
 }
 
+func (w *Watcher) doPRListRequest(ctx context.Context, username, repository string) (
+	*github.Response, []*github.PullRequest, error,
+) {
+	ctx, cancel := context.WithTimeout(ctx, RequestTimeout)
+	defer cancel()
+
+	prs, resp, err := w.c.PullRequests.List(ctx, username, repository, &github.PullRequestListOptions{})
+	if err != nil {
+		return resp, nil, err
+	}
+
+	w.updatePollInterval(resp)
```

Comment:
It looks like ' X-Poll-Interval'  is only returned by [events](https://developer.github.com/v3/activity/events/), and the [pulls](https://developer.github.com/v3/pulls) response does not include it.